app/Http/Controllers/UserController.php

```diff
@@ -81,4 +82,24 @@ public function destroy($id)
 
         return redirect()->route('user.index')->withErrors($errors);
     }
+
+    /**
+     * Search user in database
+     *
+     * @param Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function searchUser(Request $request)
+    {
+        $keyword = $request->q;
+
+        $result = User::whereRaw('`users`.`role_id` <> '
```

Comment:
[rank A] I think this query isn't complicate, so why do you want to use raw query here?